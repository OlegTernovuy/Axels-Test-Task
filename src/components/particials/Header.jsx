import React from 'react';

import { StyledHeader } from '../../styled/particials/StyledHeader';

const Header = () => (
  <StyledHeader>
    <h1>Axels Test Task</h1>
    <span>Oleh Ternovyi</span>
  </StyledHeader>
);

export default Header;
