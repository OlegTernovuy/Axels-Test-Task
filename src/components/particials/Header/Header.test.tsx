import renderer from 'react-test-renderer';
import Header from './Header';

describe('Header snapshot component', () => {
    it('Matches Snapshot', () => {
        const headerComponent = renderer.create(<Header />).toJSON();
        expect(headerComponent).toMatchSnapshot();
    });
});
