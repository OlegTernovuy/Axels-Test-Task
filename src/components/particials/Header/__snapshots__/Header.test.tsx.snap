// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot component Matches Snapshot 1`] = `
<header
  className="sc-aYaIB igTKbP"
>
  <h1>
    Axels Test Task
  </h1>
  <span>
    Oleh Ternovyi
  </span>
</header>
`;
